readme.md

```diff
@@ -193,6 +193,8 @@
 - [Statusfy](https://github.com/bazzite/statusfy) - Status page system using Tailwind CSS.
 - [Todolist](https://github.com/guillaumebriday/todolist-frontend-vuejs) - To-do list application using Tailwind CSS.
 - [LeagueStats](https://github.com/vkaelin/LeagueStats) - Statistics website for League of Legends players.
+- [SapperCommerce](https://github.com/itswadesh/sapper-ecommerce) - ecommerce using svelte & Tailwind CSS.
```

Comment:
Can you change the description to be more explicit? Ecommerce what?

Reply:
ping @tehpsalmist